.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "yarn.lock" }}
+          - v1-dependencies-
+
+      - run: yarn install
+
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "yarn.lock" }}
+
+      - run: yarn test --ci
+
+      - run: yarn semantic-release || true
```

Comment:
Doesn't this need to be `yarn run semantic-release`?

Reply:
It is possible to run script commands by [omitting the `run`](https://yarnpkg.com/lang/en/docs/cli/run/) keyword. Using `run` is more explicit though. So I will update!